lumigator/python/mzai/backend/backend/api/routes/experiments.py

```diff
@@ -60,3 +70,54 @@ def get_experiment_result_download(
     return ExperimentResultDownloadResponse.model_validate(
         service.get_job_result_download(experiment_id).model_dump()
     )
+
+
+####################################################################################################
+# "new" routes
+####################################################################################################
+# These "experiments_new" routes are not yet ready to be exposed in the OpenAPI schema,
+# because it is designed for the API when 'workflows' are supported
+# TODO: Eventually this route will become the / route,
+# but right now it is a placeholder while we build up the Workflows routes
+# It's not included in the OpenAPI schema for now so it's not visible in the docs
+@router.post("/new", status_code=status.HTTP_201_CREATED, include_in_schema=False)
+def create_experiment_id(
+    service: ExperimentServiceDep, request: ExperimentIdCreate
+) -> ExperimentResponse:
+    """Create an experiment ID."""
```

Comment:
nit: might want to also mention it's a UUID

Reply:
we'll also want to validate the experiment response as we do in other methods

Reply:
Totally! it will actually be a str, because mlflow doesn't use UUIDs, per the PR that comes after this one. https://github.com/mozilla-ai/lumigator/pull/750/files#r1930972704

I'll add the model validation wrapper 👍